content/manuals/desktop/install/windows-install.md

```diff
@@ -164,6 +164,10 @@ again when you switch back.
 
 ## Install Docker Desktop on Windows
 
+> [!TIP]
+>
+> See the [FAQs](/manuals/desktop/faqs/general.md#how-do-I-run-docker-desktop-without-administrator-privileges) on how to install and run Docker Desktop without needing administrator privileges.
```

Comment:
Non-blocking: The URL takes to the FAQs page, but not to the section on running without admin privileges